rename.sh

```diff
@@ -0,0 +1,10 @@
+current_otp="gscraper_web" 
```

Comment:
I think we don't need to add this file to the codebase

Reply:
that's right. Removed in 1425d0c.